### ISSUE TITLE
local environment missing Markup

### ISSUE THREAD

**Issue by sjsrey** [knaaptime/pysal-iale2022#1]
In the 05 nb:
```
ImportError: cannot import name 'Markup' from 'jinja2' (/Users/serge/miniconda3/envs/pysal-iale2022/lib/python3.9/site-packages/jinja2/__init__.py)
```

**Comment by knaaptime** [comment]
im heads down on this for the next couple hours. will ping shortly once everything runs through smoothly

**Comment by knaaptime** [comment]
resolved on the server (needs to ensure `jinja=3.0`)
